### PR TITLE
flawed test and check failure

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
The old method of testing checks was passing that which should fail. Revised.

### CODE REVIEW

File: tests/terraform/checks/resource/aws/test_RedShiftSSL.py [hunk 1]
```diff
@@ -1,66 +1,41 @@
...
+        self.assertEqual(summary["passed"], 2)
+        self.assertEqual(summary["failed"], 2)
+        self.assertEqual(summary["skipped"], 0)
+        self.assertEqual(summary["parsing_errors"], 0)
+
+        self.assertEqual(passing_resources, passed_check_resources)
+        self.assertEqual(failing_resources, failed_check_resources)
+
+
+if __name__ == "__main__":
```
Comment (nimrodkor) [inline tests/terraform/checks/resource/aws/test_RedShiftSSL.py:40]: @JamesWoolfenden Can you remove this?

**Review by nimrodkor — APPROVED** [review]

File: tests/terraform/checks/resource/aws/test_RedShiftSSL.py (hunk 1 above)
Comment (JamesWoolfenden) [reply tests/terraform/checks/resource/aws/test_RedShiftSSL.py:40]: which bit?

Comment (schosterbarak) [reply tests/terraform/checks/resource/aws/test_RedShiftSSL.py:40]: @nimrodkor ?

Comment (nimrodkor) [reply tests/terraform/checks/resource/aws/test_RedShiftSSL.py:40]: This line:
`if __name__ == "__main__":`

Comment (JamesWoolfenden) [reply tests/terraform/checks/resource/aws/test_RedShiftSSL.py:40]: i think that line is required. It looks like I added it but its just the layout of the difftool, i think?

**Pull request merged**